lib/desired-caps.js

```diff
@@ -0,0 +1,87 @@
+const desiredCapConstraints = {
+  platformName: {
+    presence: true,
+    isString: true,
+    inclusion: ['iOS']
```

Comment:
ah. do we want to require presence? i guess this is a broader question of whether this driver should require the full gamut of mjsonwp caps, or whether it should assume that it can only do iOS, thus the cap isn't really required...